source/guides/azure/infiniband.md

```diff
@@ -0,0 +1,493 @@
+       valid_lft forever preferred_lft forever
+3: eth1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
+    link/ether 00:15:5d:33:ff:16 brd ff:ff:ff:ff:ff:ff
+4: enP44906s1: <BROADCAST,MULTICAST,SLAVE,UP,LOWER_UP> mtu 1500 qdisc mq master eth0 state UP group default qlen 1000
+    link/ether 60:45:bd:a7:42:cc brd ff:ff:ff:ff:ff:ff
+    altname enP44906p0s2
+5: ibP59423s2: <BROADCAST,MULTICAST> mtu 4092 qdisc noop state DOWN group default qlen 256
+    link/infiniband 00:00:09:27:fe:80:00:00:00:00:00:00:00:15:5d:ff:fd:33:ff:16 brd 00:ff:ff:ff:ff:12:40:1b:80:1d:00:00:00:00:00:00:ff:ff:ff:ff
+    altname ibP59423p0s2
+```
+
+```shell
+nvidia-smi topo -m
+```
+
+```shell
+        GPU0    GPU1    GPU2    GPU3    GPU4    GPU5    GPU6    GPU7    CPU Affinity    NUMA Affinity
+GPU0    X       NV2     NV1     NV2     NODE    NODE    NV1     NODE    0-19            0
+GPU1    NV2     X       NV2     NV1     NODE    NODE    NODE    NV1     0-19            0
+GPU2    NV1     NV2     X       NV1     NV2     NODE    NODE    NODE    0-19            0
+GPU3    NV2     NV1     NV1     X       NODE    NV2     NODE    NODE    0-19            0
+GPU4    NODE    NODE    NV2     NODE    X       NV1     NV1     NV2     0-19            0
+GPU5    NODE    NODE    NODE    NV2     NV1     X       NV2     NV1     0-19            0
+GPU6    NV1     NODE    NODE    NODE    NV1     NV2     X       NV2     0-19            0
+GPU7    NODE    NV1     NODE    NODE    NV2     NV1     NV2     X       0-19            0
+
+Legend:
+
+  X    = Self
+  SYS  = Connection traversing PCIe as well as the SMP interconnect between NUMA nodes (e.g., QPI/UPI)
+  NODE = Connection traversing PCIe as well as the interconnect between PCIe Host Bridges within a NUMA node
+  PHB  = Connection traversing PCIe as well as a PCIe Host Bridge (typically the CPU)
+  PXB  = Connection traversing multiple PCIe bridges (without traversing the PCIe Host Bridge)
+  PIX  = Connection traversing at most a single PCIe bridge
+  NV#  = Connection traversing a bonded set of # NVLinks
+
+```
+
+## Install UCX-Py and tools
+
+```shell
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh
+```
+
+Accept the default and allow conda init to run. Then start a new shell.
+
+Create a conda environment (see ucx-py docs)
+
+```shell
+conda create -n ucxpy -c conda-forge -c rapidsai python=3.7 ipython ucx-proc=*=gpu ucx ucx-py dask distributed numpy cupy pytest pynvml rmm cudf dask-cudf dask-cuda -y
+...
+conda activate ucxpy
+```
+
+Clone ucx-py repo locally
+
+```shell
+git clone https://github.com/rapidsai/ucx-py.git
+cd ucx-py/benchmarks
+```
+
+## Run Tests
+
+Start by running the UCX-Py test suite, from within the `ucx-py` repo:
+
+```shell
+pytest -vs tests/
+pytest -vs ucp/_libs/tests/
+```
+
+Now check to see if IB works, for that you can run some of the benchmarks that we include in UCX-Py, for example:
```

Comment:
```suggestion
Now check to see if InfiniBand works, for that you can run some of the benchmarks that we include in UCX-Py, for example:
```

Reply:
Suggesting usage of InfiniBand in the docs to prevent any confusion, feel free to ignore this and following ones.